tests/test_unclassified_job.py

```diff
@@ -26,3 +25,23 @@ def test_open_unclassified_failure_log(base_url, selenium):
     treeherder_page.open_next_unclassified_failure()
     logviewer_page = treeherder_page.job_details.open_logviewer()
     assert logviewer_page.is_job_status_visible
+
+
+@pytest.mark.nondestructive
+def test_view_unclassified_jobs(base_url, selenium):
+    page = TreeherderPage(selenium, base_url).open()
+    all_jobs = list(page.all_jobs)
+
+    page.filter_unclassified_jobs()
+    filtered_jobs = list(page.all_jobs)
+    jobstatus = page.result_sets[0].jobs[0].title
+
+    assert not all_jobs == filtered_jobs
+    if 'testfailed' in jobstatus:
+        assert 'testfailed' in jobstatus
+
+    if 'busted' in jobstatus:
+        assert 'busted' in jobstatus
+
+    if 'exception' in jobstatus:
+        assert 'exception' in jobstatus
```

Comment:
Rather than these if:assert blocks we should do something like `assert jobstatus in ('testfailed', 'busted', 'exception')`. This is much easier to read and would fail if the job status was anything else. Currently if the job status was another value, the test would pass.